spec/codebase/codebase_local_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Codebase', codebase: true do
+  it 'does NOT offend Rubocop' do
+    expect(`rubocop --parallel --format simple`).to include 'no offenses detected'
+  end
+
+  it 'satisfies Brakeman' do
+    expect(`brakeman -w1`).not_to include '+SECURITY WARNINGS+'
+  end
+
+  it 'does NOT break zeitwerk loading' do
+    expect(`bundle exec rake zeitwerk:check`).to be_empty.or include 'All is good!'
+  end
+end
```

Comment:
Just a personal sharing: I never felt the need to run these ones in the local env: as they provide only high-level warnings/recommendations, having them only in the PR war far enough to me (when there is a warning, I know I can refactor a minor thing, but at least I'm not spending time earlier running these many times).

It's personal so feel free to keep this `codebase_local` test file ;-)

Reply:
I included this in the local test to reflect what is being tested on CI and with our current set up, Danger will comment only on changed file while having it on local help dev see the bigger picture. I think it is useful for newer devs (like me). 🙇